docs/how-to/h-config-backup-restore.md

```diff
@@ -0,0 +1,113 @@
+# Configuration backup and restore
+
+Charmed Kafka's configuration is distributed using [Charmed ZooKeeper](https://charmhub.io/zookeeper?channel=3/stable).
+A Charmed ZooKeeper backup can be stored on any S3-compatible storage.
+S3 access and configurations are managed with the [`s3-integrator` charm](https://charmhub.io/s3-integrator).
+
+This guide contains step-by-step instructions on how to deploy and configure the `s3-integrator` charm for [AWS S3](https://aws.amazon.com/s3/), send the configurations to the Charmed ZooKeeper application, and finally manage your Charmed ZooKeeper backups.
+
+## Configure `s3-integrator`
+
+First, deploy the `s3-integrator` charm:
+
+```shell
+juju deploy s3-integrator
+juju run s3-integrator/leader sync-s3-credentials access-key=<aws-access-key-id> secret-key=<aws-secret-key>
+```
+
+Then, use `juju config` to add your configuration parameters. For example:
+
+```shell
+juju config s3-integrator \
+    endpoint="https://s3.us-west-2.amazonaws.com" \
+    bucket="zk-backups-bucket-1" \
+    path="/zk-backups" \
+    region="us-west-2"
+```
+
+[note]
+The only mandatory configuration parameter in the command above is the `bucket`.
+[/note]
+
+### Integrate with Charmed ZooKeeper
+
+To pass these configurations to Charmed ZooKeeper, integrate the two applications:
+
+```shell
+juju integrate s3-integrator zookeeper
+```
+
+You can create, list, and restore backups now:
+
+```shell
+juju run zookeeper/leader create-backup
+juju run zookeeper/leader list-backups
+juju run zookeeper/leader restore backup-id=<backup-id-here>
+```
+
+## Create a backup
+
+Check that Charmed ZooKeeper deployment with configurations set for S3 storage is `active` and `idle` with the `juju status` command. Once it's active, create a backup with the `create-backup` command:
+
+```shell
+juju run zookeeper/leader create-backup
+```
+
+Charmed ZooKeeper backups created with the command above will always be **full** backups: a copy of _all_ the Charmed Kafka configuration will be stored in S3.
```

Comment:
```suggestion
Charmed ZooKeeper backups created with the command above will always be **full** backups: a copy of *all* the Charmed Kafka configuration will be stored in S3.
```
It's not an error, but let's use asterisks for consistency and compatibility. 
https://www.markdownguide.org/basic-syntax/#italic-best-practices